jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBPreparedStatement.java

```diff
@@ -181,53 +181,63 @@ public void setArray(int parameterIndex, Array x) throws SQLException {
 
   @Override
   public void setAsciiStream(int parameterIndex, InputStream x) throws SQLException {
```

Comment:
The method `setArray()` is quite similar to other changed methods, why not let `setArray()` throw `PARAMETER_SUPPORTED` too?